.github/workflows/automation-scripts/invalidate-status-checks.mjs

```diff
@@ -0,0 +1,37 @@
+// Invalidate all iTwin.js build status checks when master has a change to core/backend/package.json
+// This is specifically to catch changes to @bentley/imodeljs-native
+// This will also invalidate PRs if there's a new nightly build, however our 3 hour rule should also invalidate the same PRs
+
+import { Octokit, App } from "octokit"
+import dotenv from 'dotenv'
+
+dotenv.config();
+
+const octokit = new Octokit({
+  auth: `${process.env.GITHUB_TOKEN}`
+});
+
+let pull_requests = await octokit.request('GET /repos/{owner}/{repo}/pulls', {
```

Comment:
does this get draft prs too? we shouldnt care about those

Reply:
According to the [docs](https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests), the only options are open, closed, and all. Drafts are considered opened. I didn't realize this before, so I'll skip drafts.

Reply:
fixed on line 26